roles/nodejs/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+# nodejs version to install
+nodejs_version: "4.x"
```

Comment:
I see this var is used in some places, in other places `4.x` is hardcoded. So decide or use it everywhere or hardcode it inside the tasks.